priv/perf/README.md

```diff
@@ -2,14 +2,63 @@
 
 Umbrella app for performance/load/stress tests
 
-## How to run the test
-1. Change the urls inside the [config files](config/) for the environment you are targeting.
-1. To generate the open-api client (only need once): `make init`
-1. To run the test: `MIX_ENV=<your_env> mix test`. Or `mix test` if you want to run against local services.
+## How to run the tests
 
-### Spin up local services for development
-1. To test with local services (by docker-compose): `make start-services`
-1. To turn the local services down: `make stop-services`
+### 1. Set up the environment vars
+
+```
+export CHILD_CHAIN_URL=<childchain api url>
+export WATCHER_INFO_URL=<watcher-info api url>
+export ETHEREUM_RPC_URL=<ethereum node url>
+export CONTRACT_ADDRESS_PLASMA_FRAMEWORK=<address of the plasma framework contract>
+export CONTRACT_ADDRESS_ETH_VAULT=<address of the eth vault contract>
+export CONTRACT_ADDRESS_ERC20_VAULT=<address of the erc20 vault contract>
+export LOAD_TEST_FAUCET_PRIVATE_KEY=<faucet private key>
+```
+
+
+### 2. Generate the open-api client
+ ```
+make init
+```
+(You need to run this any time you change the env vars above)
+
+### 3. Configure the tests
+Edit the config file (e.g. `config/dev.exs`) set the test parameters e.g.
+```
+  childchain_transactions_test_config: %{
+    concurrent_sessions: 100,
+    transactions_per_session: 600,
+    transaction_delay: 1000
+  }
+```
+
+Note that by default the tests use ETH both as the currency spent and as the fee. 
+This makes the code simpler as it doesn't have to manage separate fee utxos. 
+However, if necessary you can configure the tests to use a different currency. e.g.
+```
+config :load_test,
+  test_currency: "0x942f123b3587EDe66193aa52CF2bF9264C564F87",
+  fee_amount: 6_000_000_000_000_000,
+```
+
+### 4. Run the test
+`MIX_ENV=dev mix test apps/load_test/test/load_tests/runner/childchain_test.exs`
```

Comment:
> `MIX_ENV=<your_env> mix test`. Or `mix test` if you want to run against local services.

I think original description is better though. this line is pretty specific to a single test. Unless that is what we need only. 🤷‍♂️

Or you can use this line as an example to run single test after the description of a more generalized test running instruction as this is probably the most runned single test so people can copy paste the instruction easier.